src/state/reducers/galleryReducer.js

```diff
@@ -1,21 +1,36 @@
 import { submissions } from '../actions';
 
 const initialState = {
-  WritingUrl: '',
-  PageNum: 0,
-  DrawingUrl: '',
-  children_id: 0,
+  ID: 0,
+  ChildID: 0,
+  StoryId: 0,
+  Sprint: 0,
+  EpisodeStartDate: '',
+  ModerationStatus: '',
+  HasRead: false,
+  HasWritten: false,
```

Comment:
missing "HasDrawn" property, see https://github.com/Lambda-School-Labs/scribble-stadium-be/blob/main/data/seeds/007_Submissions.js

Reply:
This is indeed missing.